packages/cli/src/commands/repo/list-deprecations.ts

```diff
@@ -45,7 +45,7 @@ export async function command(opts: OptionValues) {
 
   const deprecations = [];
   for (const [index, pkg] of packages.entries()) {
-    const results = await eslint.lintFiles(joinPath(pkg.dir, 'src'));
```

Comment:
From my perspective the timings are not so different. It looks like it takes a little bit more time since the tree of files is bigger.

````
# Without the change
❯ time yarn backstage-cli repo list-deprecations > out
yarn backstage-cli repo list-deprecations > out  199.72s user 21.13s system 149% cpu 2:27.77 total

time yarn backstage-cli repo list-deprecations > out
yarn backstage-cli repo list-deprecations > out  193.23s user 20.94s system 153% cpu 2:19.85 total

# With the change
❯ time yarn backstage-cli repo list-deprecations > out-with-change
yarn backstage-cli repo list-deprecations > out-with-change  203.28s user 22.95s system 146% cpu 2:34.75 total

❯ time yarn backstage-cli repo list-deprecations > out-with-change
yarn backstage-cli repo list-deprecations > out-with-change  186.81s user 22.06s system 144% cpu 2:24.98 total
```

I think ESLint doesn't take into account `node_modules` as it's ignored by the `.eslintignore` file, and I guess that if it was the case the timings would be a lot more different.

We could do something like this:
````
const results = await eslint.lintFiles([joinPath(pkg.dir, 'src'), joinPath(pkg.dir, 'config.d.ts')]);
```

but we need to add `errorOnUnmatchedPattern: false` to the ESLint config before. And for reasons I don't understand this makes the performance way worse:

```
❯ time yarn backstage-cli repo list-deprecations > out-with-configdts
yarn backstage-cli repo list-deprecations > out-with-configdts  392.13s user 41.83s system 126% cpu **5:43.56 total**
```

WDYT?

Reply:
@tcardonne not sure I understand the bit about `errorOnUnmatchedPattern: false`? Do we need that?